src/api/User/mutations/createUser/integration.spec.js

```diff
@@ -1,144 +1,38 @@
 import gql from 'graphql-tag'
 import testClient from '@tests/client'
 import database from '@server/database'
-import {
-  INVALID_EMAIL_ERROR,
-  EMAIL_ALREADY_IN_USE_ERROR
-} from './helpers/validateEmail'
+
+const mutation = gql`
```

Comment:
We could drop the `gql` dependency and use a `.graphql` file instead but that doesn’t change much 😄

Reply:
Writing inline GraphQL should be avoided, especially if we modify this specific type. Then again tests won't pass if this definition is wrong, so it's pretty much the same